cmd/pebble/cmd_ls.go

```diff
@@ -47,20 +55,26 @@ func (cmd *cmdLs) Execute(args []string) error {
 	opts := client.ListFilesOptions{
 		Path:    cmd.Positional.Path,
 		Pattern: "",
-		Itself:  cmd.Itself,
+		Itself:  cmd.Directory,
 	}
 	files, err := cmd.client.ListFiles(&opts)
 	if err != nil {
 		return err
 	}
 
-	fmt.Println("Name\tUser\tGroup\t")
+	w := tabwriter.NewWriter(os.Stdout, 1, 1, 1, ' ', 0)
```

Comment:
Use the package-level `Stdout` here for testability (see example in `cmd/pebble/format.go`).

Reply:
That explains why my `s.Stdout()` assertions didn't work, thank you!